Framework/AnalysisSupport/test/test_DataInputDirector.cxx

```diff
@@ -16,7 +16,8 @@
 #include <boost/test/unit_test.hpp>
 
 #include "Headers/DataHeader.h"
-#include "Framework/DataInputDirector.h"
+// TODO how to write this better?
+#include "../src/DataInputDirector.h"
```

Comment:
Somehow I don't manage to improve this even with `PRIVATE_INCLUDE_DIRECTORIES ${CMAKE_CURRENT_LIST_DIR}/src`
Could you advise?

Reply:
I think this is fine, frankly.